lib/internal/bootstrap_node.js

```diff
@@ -7,9 +7,13 @@
 
 'use strict';
 
-(function(process) {
+(function(global, process) {
+
+  function startup(global, process) {
+    // Expose the global object as a property on itself
+    // (Allows you to set stuff on `global` from anywhere in JavaScript.)
+    global.global = global;
```

Comment:
@thefourtheye Have you seen https://github.com/nodejs/node/pull/10405 / https://tc39.github.io/proposal-global/?